pkg/preparer/orchestrate.go

```diff
@@ -56,6 +56,18 @@ func (p *Preparer) WatchForHooks(quit chan struct{}) {
 	}
 }
 
+// Sync the hooks just once. This is useful at preparer startup so that hooks
+// will be synced before normal pod management begins
+func (p *Preparer) SyncHooksOnce() {
+	hookErrCh := make(chan error)
```

Comment:
any number of hooks can fail to be installed, i thought we should mimic the behavior of the normal loop